eagleEvents/templates/event.html.j2

```diff
@@ -0,0 +1,48 @@
+{% extends "layout.html.j2" %}
+{% block head_content %}
+    <title>Eagle Events Event</title>
+{% endblock %}
+    <h1>Events</h1>
```

Comment:
Missing a start block for the body content block

Reply:
Added a start block for the body content block to the event page and also fixed the table header.